README.md

```diff
@@ -5,7 +5,7 @@
 
 ![text](https://user-images.githubusercontent.com/74983536/128005620-f7f44ba5-14d7-4329-abe4-0da9c93137e1.png)
 
-THE HARRY POTTER GAME:
+### **THE HARRY POTTER GAME:**
```

Comment:
I think the header h3 (`###`) already makes the text bold, so the `**` is not needed

Reply:
Yeah sure!